setup/apache.yml

```diff
@@ -28,6 +28,10 @@ apache_terminate_ssl:
   global: apache_terminate_ssl
   default: yes
 
+apache_staging_prefix:
+  global: apache_staging_prefix
+  default: /sysadmin/dist-www
```

Comment:
If this is the default value, the existing functionality will have to explicitly set the apache_staging_prefix to `''` ? or `'/'` ?